tests/integration/src/test/java/org/apache/pulsar/tests/integration/presto/TestBasicPresto.java

```diff
@@ -0,0 +1,47 @@
+package org.apache.pulsar.tests.integration.presto;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.pulsar.tests.integration.docker.ContainerExecResult;
+import org.apache.pulsar.tests.integration.topologies.PulsarCluster;
+import org.apache.pulsar.tests.integration.topologies.PulsarClusterTestBase;
+import org.testng.ITest;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+public class TestBasicPresto extends PulsarClusterTestBase implements ITest {
+
+    @BeforeSuite
+    public void setupCluster() throws Exception {
+        super.setupCluster();
+        ContainerExecResult containerExecResult = pulsarCluster.getBroker(0).execCmd(PulsarCluster.PULSAR_COMMAND_SCRIPT, "sql-worker", "start");
+        log.info("SQL worker startup output {}", containerExecResult.getStdout());
+        assertThat(containerExecResult.getExitCode()).isEqualTo(0);
+        log.info("Sleep for sql worker initialization");
+        TimeUnit.MINUTES.sleep(1);
```

Comment:
explain why do we need this? or can we provide a more deterministic way to check whether sql worker is ready to be used? e.g. check http endpoint or tcp port